demos/bunny/wasmmark/src/main.zig

```diff
@@ -105,81 +100,83 @@ fn addBunny() void {
 }
 
 fn removeBunny() void {
-    if (bunnyCount==0) return;
+    if (bunnyCount == 0) return;
 
     bunnyCount -= 1;
 }
 
 export fn testscreen() void {
-    var i : usize = 0;
-
-    while (i<2000) {
-    // tic.ZERO.* = 0x99;
-    tic.FRAMEBUFFER[i]=0x56;
-    // tic.FRAMEBUFFER2.*[i]=0x67;
-    // tic.ZERO[i]= 0x56;
-    // bunnies[i].draw();
-    i += 1;
+    var i: usize = 0;
+
+    while (i < 2000) {
+        // tic.ZERO.* = 0x99;
+        tic.FRAMEBUFFER[i] = 0x56;
+        // tic.FRAMEBUFFER2.*[i]=0x67;
+        // tic.ZERO[i]= 0x56;
+        // bunnies[i].draw();
+        i += 1;
     }
 }
 
 export fn BOOT() void {
-    rnd = RndGen.init(0).random();
+    var xoshiro = RndGen.init(0);
+    rnd = xoshiro.random();
     fps.initFPS();
     addBunny();
 }
 
 export fn TIC() void {
-    if (t==0) {
+    if (t == 0) {
         tic.music(0, .{});
     }
-    if (t == 6*64*2.375) {
+    if (t == 6 * 64 * 2.375) {
         tic.music(1, .{});
     }
     t = t + 1;
 
-
-if (tic.btn(0)) {
-    var i : i32 = 0;
-    while (i<5) {
-        addBunny();
-        i+=1;
+    if (tic.btn(0)) {
+        var i: i32 = 0;
+        while (i < 5) {
+            addBunny();
+            i += 1;
+        }
     }
-}
 
-if (tic.btn(1)) {
-    var i : i32 = 0;
-    while (i<5) {
-        removeBunny();
-        i+=1;
+    if (tic.btn(1)) {
+        var i: i32 = 0;
+        while (i < 5) {
+            removeBunny();
+            i += 1;
+        }
     }
-}
 
-// 	-- Update
-var i : u32 = 0;
-while (i<bunnyCount) {
-    bunnies[i].update();
-    i += 1;
-}
-
-// 	-- Draw
-tic.cls(15);
-i = 0;
-while (i<bunnyCount) {
-    bunnies[i].draw();
-    i += 1;
-}
+    // 	-- Update
+    var i: u32 = 0;
+    while (i < bunnyCount) {
+        bunnies[i].update();
+        i += 1;
+    }
 
+    // 	-- Draw
+    tic.cls(15);
+    i = 0;
+    while (i < bunnyCount) {
+        bunnies[i].draw();
+        i += 1;
+    }
 
-tic.rect(0, 0, screenWidth, toolbarHeight, 9);
+    tic.rect(0, 0, screenWidth, toolbarHeight, 9);
 
-_ = tic.printf("Bunnies: {d}", .{bunnyCount}, 1, 0, .{.color = 11});
-_ = tic.printf("FPS: {d:.4}", .{fps.getValue()}, screenWidth / 2, 0, .{.color = 11});
-_ = tic.print("hello people", 10, 10, .{.color = 11});
+    _ = tic.printf("Bunnies: {d}", .{bunnyCount}, 1, 0, .{ .color = 11 });
+    _ = tic.printf("FPS: {d:.4}", .{fps.getValue()}, screenWidth / 2, 0, .{ .color = 11 });
+    _ = tic.print("hello people", 10, 10, .{ .color = 11 });
 
-// var x : u32 = 100000000;
-// tic.FRAMEBUFFER[x] = 56;
+    // var x : u32 = 100000000;
+    // tic.FRAMEBUFFER[x] = 56;
 
-// testscreen();
+    // testscreen();
 }
 
+export fn BDR() void {}
+
+export fn OVR() void {}
```

Comment:
Are these necessary now to compile?

Reply:
yep, if these are not present then the Zig compiler will complain that it couldn't find the symbols we're telling it to export.